app/javascript/components/journey/solutions-list/HeadTestsStatusSelect.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { SingleSelect } from '../../common'
+
+export type HeadTestsStatus =
+  | undefined
+  | 'passed'
+  | 'failed'
+  | 'errored'
+  | 'exceptioned'
```

Comment:
Do we want to have individual states, or just a `passed`/`failed` option?

And what do we do with the `not_queued`, `queued` and `cancelled` states?

Reply:
For now, I'm happy with these individual states I think.